pcap/build.gradle

```diff
@@ -1,10 +1,11 @@
 plugins {
     id 'java-library'
+    id "biz.aQute.bnd.builder" version "7.0.0"
 }
 
 java {
     toolchain {
-        languageVersion = JavaLanguageVersion.of(11)
+        languageVersion = JavaLanguageVersion.of(21)
```

Comment:
PCAP requires java 11, not java 21. Pcap library does not need java 21, so it shouldn't update, so you don't end up with backwards compatibility issues! Studio V5 includes pcap, and Studio V5 does not use java 21.

So don't upgrade java if you don't need the actual features from java 21!